src/SFA.DAS.EmployerFinance.UnitTests/SFA.DAS.EmployerFinance.UnitTests.csproj

```diff
@@ -1,13 +1,15 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net462</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
Can the following elements be added?

```
<LangVersion>latest</LangVersion>
<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Reply:
They're already in master 🙂

Reply:
I've merge master and now they have been added